static/src/stylesheets/module/experiments/_reading-time.scss

```diff
@@ -0,0 +1,141 @@
+#reading-time.fc-container--thrasher {
+    background: $guardian-brand-dark;
+    .fc-item {
+        background: $guardian-brand;
+    }
+}
+
+.reading-time__time-selection {
+    @include button-colour(
+            $news-main-1, // fill
+            #ffffff, // text
+            $news-main-2 //border
+    );
+    @include button-hover-colour(
+            darken($news-main-1, 5%),
+            $news-main-2
+    );
+
+    font-weight: normal;
+    margin-bottom: 5px;
+}
+.reading-time__wrapper {
+    position: relative;
+    @include mq(desktop) {
+        width: 100%;
+    }
+    .text-selection-block {
+        .cta {
+            line-height: 1.3em;
+            font-weight: bold;
+            vertical-align: top;
+            padding-top: 5px;
+        }
+
+        .cta, .selections {
+            width: 45%;
+            display: inline-block;
+            padding-bottom: 15px;
+        }
+
+        .btn {
+            display: inline-block;
+            vertical-align: top;
+            position: relative;
+            top: 1px;
+            line-height: 30px;
```

Comment:
This might be OK for a test, but generally you would try and use variables & multiples of variables for all of these - `$gs-baseline` for vertical height, `gs-span` for horizontal widths. This is for design consistency and comprehensibility of the code. Also for font-related stuff there are some typography helpers. Have a look in the `pasteup` directory for all of this